packages/template-retail-react-app/app/ssr.js

```diff
@@ -41,6 +41,7 @@ const {handler} = runtime.createHandler(options, (app) => {
                 directives: {
                     'img-src': ["'self'", '*.commercecloud.salesforce.com', 'data:'],
                     'script-src': ["'self'", "'unsafe-eval'", 'storage.googleapis.com'],
+                    'connect-src': ["'self'", 'api.cquotient.com'],
```

Comment:
Double checking that there are zero _other_ `ssr.js` files that need this change

Reply:
Thanks for going over this PR @bfeister . 

This should be the only ssr.js file that needs the change. All other ssr.js files are either very barebones (typescript-minimal and express-minimal) or are used for development and do not call Einstein.